scripts/stitching/stitching.py

```diff
@@ -0,0 +1,325 @@
+""" Script for creating a stitched ROI MIBItiff from a folder of FOVs.
+"""
+
+import argparse
+import json
+import os
+import sys
+import time
+
+import numpy as np
+
+from mibidata import mibi_image as mi
+from mibidata import tiff
+from mibitracker.request_helpers import MibiRequests
+
+MAX_TRIES = 5
```

Comment:
Can we rename this and have a comment on what this is used for? Something like `MAX_MIBITRACKER_REST_TRIES` or something